app/utils.py

```diff
@@ -0,0 +1,50 @@
+from selenium import webdriver
+from selenium.common import NoSuchElementException
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from webdriver_manager.chrome import ChromeDriverManager
+
+
+class CustomDriver:
+    class __CustomDriver:
+        def __init__(self):
+            self.options = Options()
+            self.options.add_argument("--headless")
+            self.driver = webdriver.Chrome(
+                service=Service(ChromeDriverManager().install()),
+                options=self.options,
+            )
+
+    driver = None
+
+    def __init__(self):
+        if not self.driver:
+            CustomDriver.driver = CustomDriver.__CustomDriver().driver
+
+    def get(self, url: str):
+        self.driver.get(url)
+        self.accept_cookie()
+
+    def click_more(self):
+        try:
+            btn_more = self.driver.find_element(
+                By.CLASS_NAME, "ecomerce-items-scroll-more"
+            )
+            while btn_more.is_displayed():
+                self.driver.execute_script("arguments[0].click()", btn_more)
+
+        except NoSuchElementException:
+            pass
+
+    def close(self):
+        self.driver.close()
+
+    def accept_cookie(self) -> None:
+        try:
+            self.driver.find_element(By.CLASS_NAME, "acceptCookies").click()
+        except NoSuchElementException:
+            pass
```

Comment:
add the info for dev instead of just "pass"

Reply:
It is not necessary there, because it will break the information that is displayed and has not any sense for dev

Reply:
`pass` in except blocks is bad practice. If you catch an exception, you must write information about it to log. It makes the life of other developers easy.